Expensify/App
# Do not mark chats as read unless you are actually in the chat

@expensifyrandomdrake 🐻 
cc @tgolen 

### Fixed Issues
Fixes $https://github.com/Expensify/Expensify/issues/143014

### Tests
1. In the web chat, open a conversation with a second user (User 2), then move to a different tab in the browser (without leaving the conversation in Native Chat).
2. Now, add a message to the chat from User 2. You can either open another web chat in incognito mode or user the desktop/mobile versions.
3. You should get a notification (for User 1):
<img width="364" alt="Screen Shot 2020-10-15 at 12 26 02 PM" src="https://user-images.githubusercontent.com/19366280/96111559-a4c90500-0ee1-11eb-8a30-765474a4a8f1.png">
4. And the tab itself should also have a new message notification:
<img width="201" alt="Screen Shot 2020-10-15 at 12 27 23 PM" src="https://user-images.githubusercontent.com/19366280/96111669-cde99580-0ee1-11eb-860f-5ad21c6ea16a.png">
5. Make sure the tab notification doesn't go away on its own.
6. Open the Native Chat tab. The chat that received the message should be highlighted:
<img width="307" alt="Screen Shot 2020-10-15 at 12 28 47 PM" src="https://user-images.githubusercontent.com/19366280/96111809-00938e00-0ee2-11eb-9651-fa7f55b96944.png">
7. After 3 seconds, both the highlight and the tab notification should go away, effectively making the chat as "read".
8. This change only really matters in browser flows, but do some regression testing on the other platforms to make sure everything still works as expected.
